src/crimson/os/seastore/cache.cc

```diff
@@ -394,6 +394,35 @@ Cache::replay_delta(
   }
 }
 
+Cache::get_next_dirty_extents_ret Cache::get_next_dirty_extents(
+  journal_seq_t seq)
+{
+  std::vector<CachedExtentRef> ret;
```

Comment:
how about adding 
```c++
assert(std::is_sorted(dirty.begin(), dirty.end(),
                      [](const CachedExtent& lhs, const CachedExtent& rhs) {
                        return lhs.dirty_from < rhs.dirty_from;
                      }));
```
here?

Reply:
I don't want to traverse the whole list here, but I will add an assert validating the entries we do look at.